geopackage_validator/gdal_utils.py

```diff
@@ -50,3 +51,19 @@ def check_gdal_installed():
         sys.exit(
             "ERROR: cannot find GDAL/OGR modules, follow the instructions in the README to install these."
         )
+
+
+@lru_cache(None)
+def dataset_geometry_types(dataset):
```

Comment:
Is dit wel een gdal util, en zou je die util dan ook niet gebruiken in de validaties die ook de types nodig hebben

Reply:
hier ben ik het mee eens. Ik kijk even.

Reply:
deze gdal_util wordt sowieso hernoemt naar util in een andere PR.